prac_05/word_occurences.py

```diff
@@ -0,0 +1,17 @@
+"""
+Word Occurences
+Estimate: 15 minutes
+Actual: 11 minutes
+"""
+
+word_to_count = {}
+user_string = input("Text: ").lower().split()
+for word in user_string:
+    try:
+        word_to_count[word] = word_to_count[word] + 1
+    except KeyError:
+        word_to_count[word] = 1
+print(word_to_count)
+words = sorted(list(word_to_count.keys()))
+for word in words:
+    print(f"{word} : {word_to_count[word]}")
```

Comment:
A separate task asks for formatting that aligns all ':' with one another. Similar to: 
a**********: 2
collection : 1      
fun*******: 1
(I had to use * as empty spaces are removed automatically)
A good way to do this is with the variable "max_width = max(len(word) for word in words)"